### PR TITLE
Added examples of quick-auth

### PR DESCRIPTION
**Pull request by Diego-Romero**
## JIRA Ticket

https://atlassian.thetradedesk.com/jira/browse/OPENPASS-1063

## What does this PR do?

- Adds examples of how to use quick auth on all of its iterations

## Where should the reviewer start?

- check the documentation is right
- run it locally and make sure it works

## Test plan

- Manual test

## Screenshots / Videos

https://github.com/openpass-sso/openpass-sample-sites/assets/10360735/92eed885-4b35-45ee-80ea-47ac5234dccc

### CODE REVIEW

File: openpass-js-sdk/src/views/quick-auth-html.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    document.getElementById("show-quick-auth").addEventListener("click", () => {
+        oneTapDiv.setAttribute("data-openpass-quick-auth-show", "true");
+    });
+
+    document.getElementById("hide-quick-auth").addEventListener("click", () => {
+        oneTapDiv.setAttribute("data-openpass-quick-auth-show", "false");
+    });
+
+    function popupSuccessCallback(signInResponse) {
+        oneTapDiv.setAttribute("data-openpass-quick-auth-show", "false");
```
Comment (christriddle-thetradedesk) [inline openpass-js-sdk/src/views/quick-auth-html.html:56]: Should we display something in the UI here?

File: openpass-js-sdk/src/views/quick-auth-popup.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    document.getElementById("clear-session").addEventListener("click", () => {
+        localStorage.removeItem("openpass-authenticated");
+        window.location.reload();
+    });
+
+    function popupSuccessCallback(signInResponse) {
+
+        localStorage.setItem("openpass-authenticated", true)
+        console.log("quick-auth: Popup flow sign in succeeded, email address: " + signInResponse.idToken.email, " setting session storage to true");
```
Comment (christriddle-thetradedesk) [inline openpass-js-sdk/src/views/quick-auth-popup.html:50]: Ditto above

File: openpass-js-sdk/src/views/quick-auth-redirect.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    document.getElementById("show-quick-auth").addEventListener("click", () => {
+        authClient.show();
+    });
+
+    document.getElementById("hide-quick-auth").addEventListener("click", () => {
+        authClient.hide();
+    });
+
+    function popupSuccessCallback(signInResponse) {
+        console.log("quick-auth: Popup flow sign in succeeded, email address: " + signInResponse.idToken.email, " setting session storage to true");
```
Comment (christriddle-thetradedesk) [inline openpass-js-sdk/src/views/quick-auth-redirect.html:69]: Ditto above

File: openpass-js-sdk/src/views/quick-auth-html.html (hunk 1 above)
Comment (Diego-Romero) [reply openpass-js-sdk/src/views/quick-auth-html.html:56]: Whoops, I didn't realise that we were for the other popup examples, yep, let me update these!

File: openpass-js-sdk/src/views/quick-auth-popup.html (hunk 2 above)
Comment (Diego-Romero) [reply openpass-js-sdk/src/views/quick-auth-popup.html:50]: ditto :)

**Pull request merged**